plugins/@grouparoo/iterable/src/lib/export/listMethods.ts

```diff
@@ -0,0 +1,85 @@
+import { objectCache } from "@grouparoo/core";
+
+async function getLists(client, appGuid, appOptions, update = false) {
+  const cacheDurationMs = 1000 * 60 * 10; // 10 minutes
+  const cacheKey = ["getLists", appOptions];
+  const read = !update; // if updating, skip the read from cache. still write.
+  return objectCache(
+    { objectGuid: appGuid, cacheKey, cacheDurationMs, read },
+    async () => {
+      return client.lists.get();
+    }
+  );
+}
+
+async function getListId(
+  client,
+  appGuid,
+  appOptions,
+  groupName
+): Promise<string> {
+  groupName = (groupName || "").toString().trim();
+  if (groupName.length === 0) {
+    return `Iterable empty groupName`;
```

Comment:
not sure about this.

Reply:
Not sure if the other plugins handle this. it's a good idea. --- let's try something more user facing like `"Grouparoo Default"`